tests/test_parser.py

```diff
@@ -177,3 +177,11 @@ async def test_dynamic_js_parsing():
     # Check if the content contains the expected information
     should_contain_info = "6 Types of experimentation"
     assert should_contain_info.lower() in results.strip().lower()
+
+
+@pytest.mark.asyncio
+async def test_table_parsing():
+    sample = "examples/inputs/test_1.pdf"
+    parser_type = "AUTO"
```

Comment:
Shouldn't the test be for `STATIC_PARSE` or both?

Reply:
Ah yes, it should be `STATIC_PARSE`. Will fix.

Reply:
Done